src/atoms/SubmitButton/SubmitButton.css

```
@@ -0,0 +1,27 @@
+.submit-button{
```

```suggestion
.submit-button {
```

Revise a indentação desse arquivo!